python/llm/example/CPU/PyTorch-Models/Save-Load/README.md

```diff
@@ -0,0 +1,50 @@
+# BigDL-LLM Save/Load Low-Bit Model
+
+In this example, we show how to save/load model with BigDL-LLM low-bit optimizations (including INT8/INT5/INT4), and then run inference on the optimized low-bit model.
+
+## 0. Requirements
+To run this example with BigDL-LLM, we have some recommended requirements for your machine, please refer to [here](../../README.md#system-support) for more information.
+
+## Example: Predict Tokens using `generate()` API
+In the example [generate.py](./generate.py), we show a basic use case of low-bit optimizations on a Llama2 model to predict the next N tokens using `generate()` API, with BigDL-LLM low-bit optimizations.
```

Comment:
I think we should update descriptions about this example. And we could mention that saving and loading operations are platform-independent.

Reply:
Already done as you said.